pkgs/applications/virtualization/spike/default.nix

```diff
@@ -2,26 +2,18 @@
 
 stdenv.mkDerivation rec {
   pname = "spike";
-  version = "1.0.0";
+  version = "1.1.0";
 
   src = fetchFromGitHub {
     owner = "riscv";
     repo = "riscv-isa-sim";
     rev = "v${version}";
-    sha256 = "1hcl01nj96s3rkz4mrq747s5lkw81lgdjdimb8b1b9h8qnida7ww";
+    sha256 = "sha256-4D2Fezej0ioOOupw3kgMT5VLs+/jXQjwvek6v0AVMzI=";
   };
 
   nativeBuildInputs = [ dtc ];
   enableParallelBuilding = true;
 
-  patches = [
-    # Add missing headers to fix build.
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Sorry， I forgot it. 
Removed